### ISSUE TITLE
Add device names from ports when ports are named (e.g. HDMI)

### ISSUE THREAD

**Issue by C0rn3j** [saivert/pwvucontrol#41]
See this PR: https://gitlab.freedesktop.org/pulseaudio/pavucontrol/-/merge_requests/102 and its associated issue.

EDIT: The mentioned problem with mangled names is exclusive to Nvidia HDMI, PR for it is here https://github.com/NVIDIA/open-gpu-kernel-modules/pull/715

It should also be fixed downstream in future Pulse/PipeWire versions - https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/issues/3831

Though for the immediate future I suppose pwvucontrol should also strip them down-downstream.

**Comment by saivert** [comment]
I have implemented this now in the main branch. It only shows up for the port dropdown on the actual output devices.

The sound card profiles does not have this kind of metadata in pipewire. I would have to do correlation between profiles and routes ala. what GNOME control center does. If someone wants to contribute this I would be real happy.

**Comment by C0rn3j** [comment]
> I have implemented this now in the main branch. It only shows up for the port dropdown on the actual output devices.

I appreciate that, although the main place where I'd want to see it is in the Cards tab, could you keep the issue open until it's sorted there too or should I open a new one?

Would it make sense to ask Pipewire to add the data to the card profiles?

**Comment by saivert** [comment]
Yes it must be added to the card profiles themselves but I'm not sure if that always makes sense. HDMI device names are in relation to the actual ports typically, not a given sound card's profile.

**Comment by C0rn3j** [comment]
![Image](https://github.com/user-attachments/assets/cd7f8ed7-28ca-42b3-b6e6-d97157c0b310)

![Image](https://github.com/user-attachments/assets/1a27aeb3-2b18-4de8-a81b-1816c5e3976a)

On my systems it always bundles all my ports under one Card, which makes it extremely cumbersome without names
